torch/fx/experimental/proxy_tensor.py

```diff
@@ -376,7 +462,7 @@ def run(self, *args, **kwargs):
         with decompose(self.decomposition_table):
             return super().run(*args, **kwargs)
 
-def make_fx(f, decomposition_table=None, trace_factory_functions=True, use_fake=False):
+def make_fx(f, decomposition_table=None, trace_factory_functions=True, use_fake=True):
```

Comment:
We should do this switch in a prep PR, I think

Reply:
Ah I'll keep this off in this PR.